src/initializeApp.js

```diff
@@ -45,6 +45,7 @@ export default async function init(urlParams, env = process.env) {
 
   setOptions(urlParams, store)
 
+  await store.dispatch(`preloadNetworkCapabilities`) // wait, until networks' data is loaded
```

Comment:
why wait?

Reply:
Otherwise, we will have missing networks in some functions

Reply:
Like it's now))

Reply:
I think you should expect and handle asynchronicity with most data.

Reply:
Why? It's a pain) Routes are expecting networks. checkForPersistedNetwork too.